src/components/SignName.vue

```diff
@@ -137,6 +146,16 @@ export default {
     },
 
     methods: {
+      checkForm() {
+      const requiredFields = document.querySelectorAll('input[required], textarea[required]');
```

Comment:
I think that this could be simplified by using the `form` computed property that already exists. For example
```
for ([formValue] in Object.values(this.form)) {
  if (!formValue) {
    // some logic
    }
}
```
This may not work exactly, but just a suggestion.